### ISSUE TITLE
Microsoft Outlook does not support RDATE without RRULE

### ISSUE THREAD

**Issue by EricTRL** [esrg-knights/Squire#214]
According to [Microsoft's specification for the RDATE-property](https://docs.microsoft.com/en-us/openspecs/exchange_server_protocols/ms-oxcical/ed2f4b82-2803-4232-a3ba-253898a9885d):
> If this property is specified, an RRULE MUST also be specified in the same VEVENT.

Attempting to import such events into Outlook causes Outlook to issue a warning and ignore the entire event.

This was a problem for the following (past) activities: Extra Boardgame Evening, Knights Study Session

Note that this goes directly against RFC 2445 referenced in their docs, which states the following for an RDATE:
> This property can appear along with the "RRULE" property
   to define an aggregate set of repeating occurrences. When they both
   appear in an iCalendar object, the recurring events are defined by
   the union of occurrences defined by both the "RDATE" and "RRULE".

**Comment by DutcherNL** [comment]
The current version of the calendar system has little interaction with RDATE. Any additional instance created under the Activity that is not part of the recurrence is given a unique identifier outside of the identifier of the activity to prevent any miscommunication with external calendar behaviours related to recurring instances.

That said, it is still possible to create RDATEs in the admin for which the system will react accordingly with a related activity at that instance (if I'm not mistaken). However it is not expected to be used often.

**Comment by EricTRL** [comment]
I agree that RDATEs are currently not being used a lot anymore right now. I think we should phase out RDATEs completely eventually. The same would go for EXDATEs now that we have #219. Doing so would also mean we can remove a lot of the weird code that was required to handle problems with the recurrence-library.
